storage/engine/mvcc_test.go

```diff
@@ -1157,6 +1158,23 @@ func TestMVCCDeleteRangeConcurrentTxn(t *testing.T) {
 	}
 }
 
+// TestMVCCDeleteRangeOldTimestampNoValue verifies that
+// MVCCDeleteRange does not return an error if there is a write
+// intent, but the key did not exist at the specified timestamp.
+func TestMVCCDeleteRangeOldTimestampNoValue(t *testing.T) {
+	defer leaktest.AfterTest(t)
+	engine := createTestEngine()
+	defer engine.Close()
+
+	err := MVCCPut(engine, nil, testKey1, makeTS(1, 0), value1, nil)
```

Comment:
sorry to nit: would you mind making these  `err`s block scoped? i.e. `if _, err := Blah(); err != nil {`